packages/checkbox-react/example/index.tsx

```diff
@@ -1,4 +1,5 @@
-import "@fremtind/jkl-checkbox/checkbox.css";
+import "@fremtind/jkl-checkbox/checkbox.min.css";
+import "@fremtind/jkl-field-group/field-group.min.css";
 import { initTabListener } from "@fremtind/jkl-core";
 import "@fremtind/jkl-core/core.css";
 import React from "react";
```

Comment:
Does this not annoy your ocd? I can feel it.

Reply:
Haha!